pegasus/sites.v3/hourofcode.com/styles/060-hoc-specific.css

```diff
@@ -224,3 +224,34 @@ iframe {
   height: 1px;
   overflow: hidden;
 }
+#hoc-event-location-selector .mapboxgl-ctrl-geocoder {
```

Comment:
This is the stuff I ended up doing as well. It's kind of awful but I couldn't think of a better way either :/